### ISSUE TITLE
How can I import library in a typescript file?

### ISSUE THREAD

**Issue by SaeedZhiany** [joltup/rn-fetch-blob#138]
Is it possible to explain how can I use this library in a typescript based project? It seems that @types is not defind for this project, because `npm install @types/rn-fetch-blob` does not work. also I don't know how can I use the `types.js` to import types.

currently I get this IDE error:

`TS7016: Could not find a declaration file for module 'rn-fetch-blob'. 'C:/Users/SAEED/WebstormProjects/reactNative/node_modules/rn-fetch-blob/index.js' implicitly has an 'any' type.   Try `npm install @types/rn-fetch-blob` if it exists or add a new declaration (.d.ts) file containing `declare module 'rn-fetch-blob';`

**Comment by nomoreboredom** [comment]
You can just silence that rule, the plugin you are using for linting should have some settings on your IDE

**Comment by SaeedZhiany** [comment]
when silence the rule, I just ignore the problem, am I right?
I don't want to do this. I want to use typescript advantages.

**Comment by JensDebergh** [comment]
@SaeedZhiany I'm experiencing the same problem. 

The reason why `npm install @types/rn-fetch-blob` does not work is because it hasn't been made yet.

There was a typescript definition file available for `react-native-fetch-blob` but not for the fork `rn-fetch-blob`.

I'm afraid we will have to create the typescript defintion file ourselves.

**Comment by Traviskn** [comment]
Yeah sorry I don't know anything about typescript, so adding typescript definitions is something we will need someone with a better understanding of typescript to submit a PR for

**Comment by reilem** [comment]
PR made for this #184

**Comment by jdgray** [comment]
@Traviskn I manually added the index.d.ts for now because the file is in master, but for some reason npm is not pulling it down.

**Comment by vreality64** [comment]
@jdgray 
The Latest release version is `0.10.13` currently which doesn't have `index.d.ts`. That's the reason file not exist 😃
